Dockerfile

```diff
@@ -36,5 +34,3 @@ FROM base
 COPY --from=build \
     /usr/src/ccconsensus/target/release/ccconsensus \
     /usr/local/bin/ccconsensus
-
-CMD ["ccconsensus", "-v"]
```

Comment:
And how about this line ?  Removing it will cause the containers to not have a default entry point, i.e. that always needs to be specified. Is this what we really want ? I find it strange for container images to not have a default entry point and require the user to manually specify one.

Reply:
I noticed a build pattern. Dockerfiles for the [validator](https://github.com/gluwa/Sawtooth-Core/blob/dev/validator/Dockerfile), the [gateway](https://github.com/gluwa/Creditcoin-Gateway/blob/dev/ccgateway/Dockerfile) and [processor](https://github.com/gluwa/Creditcoin-Processor-Rust/blob/dev/Dockerfile) do not have entry points and they rely on the docker-compose for an entry point. I'm not familiar with the best practices, if you feel we should have it done differently, let's change it.

Reply:
I can't say I'm familiar with some official best practice but IMO from a usage stand point it is easier to just execute `docker run <image>` rather than specifying additional commands and options. 

The policy here IMO should be "if running the default command results in a functioning application/container then that should be the default entry point". Power users can change that very easily by specifying their own entry point, this also will not affect existing Dockerfile files. 

So the question here would be "Can the consensus auto-discover the rest of the components in the sawtooth network and communicate with them". If the answer above is Yes (in the context of our docker-compose) then keep the command.